packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -8,6 +8,20 @@ import 'package:flutter/painting.dart';
 
 export 'dart:ui' show TextDirection;
 
+/// Determines the assertiveness level of the accessibility announcement.
+///
+/// It is used to set the priority with which assistive technology should treat announcements.
+enum Assertiveness {
+  /// The assistive technology will speak changes whenever the user is idle.
```

Comment:
How does the assistive technology know that the user is idle? 😝 Isn't this more about no other announcements currently being in progress?

Reply:
This [documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) explains `polite` as:

> Normally, only aria-live="polite" is used. Any region which receives updates that are important for the user to receive, but not so rapid as to be annoying, should receive this attribute. The screen reader will speak changes whenever the user is idle.

[This](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) goes into a bit more detail:

> The aria-live attribute is the primary determination for the order of presentation of changes to live regions. Implementations will also consider the default level of politeness in a [role](https://www.w3.org/TR/wai-aria/#dfn-role) when the aria-live attribute is not set in the ancestor chain (e.g., [log](https://www.w3.org/TR/wai-aria/#log) changes are polite by default). Items which are assertive will be presented immediately, followed by polite items. User agents or assistive technologies MAY choose to clear queued changes when an assertive change occurs. (e.g., changes in an assertive region may remove all currently queued changes)
When live regions are marked as polite, assistive technologies SHOULD announce updates at the next graceful opportunity, such as at the end of speaking the current sentence or when the user pauses typing. When live regions are marked as assertive, assistive technologies SHOULD notify the user immediately. Because an interruption may disorient users or cause them to not complete their current task, authors SHOULD NOT use the assertive value unless the interruption is imperative.

We could change the wording to make it more clear. My understanding is that it's not just about other announcements being in progress, but it also considers if user is actively working on a task.

Reply:
If the official documentation also phrases it like that I am ok with keeping it as-is.